src/v0/destinations/user/transform.js

```diff
@@ -57,7 +63,7 @@ const identifyResponseBuilder = async (message, destination) => {
 
 const trackResponseBuilder = async (message, destination) => {
   if (!message.event) {
-    throw new CustomError("[ User.com ]:: parameter event is required", 400);
+    throw new InstrumentationError("[User.com]:: parameter event is required");
```

Comment:
There is no need to add the destination name in the error messages.
```suggestion
    throw new InstrumentationError("parameter event is required");
```

Reply:
updated